### DISCUSSION TITLE
Possible to export and import schemas within different pocketbase projects?

### DISCUSSION THREAD

**Discussion by zeshhaan** [pocketbase/pocketbase#5567] [Q&A] [answered]
Hi, I'm new to pocketbase and just exploring some use-cases.
Usually when working with client projects of same niche, I want to use the exact same schema of previous project. 
Is there a way to export and import (just the schema) from one pocketbase project to another?

(I have found somewhat similar discussion here https://github.com/pocketbase/pocketbase/discussions/5549 but it didn't answer my question, so asked here separately. i hope thats fine)

**Comment by lumosminima** [comment, marked as answer]
Hi there, I asked #5549 and my use case was creating a subset db from app db for data export purpose and because I didn't find some function directly returning the json schema and gani mentioned migration, I just used the generated code from migration using: `go run main.go migrate collections` and then just took the jsonData string since I only needed to create all the tables in the new db automatically by passing this to `dao.ImportCollections`. But for your usecase if you are not extending with go there is built-in import/export of collections schema and even data with backups in admin-ui and if you want this programmatically then you can do what I mentioned above.

**Reply by zeshhaan** [reply]
thanks! works for me
